crate_universe/private/splicing_utils.bzl

```diff
@@ -155,6 +159,12 @@ def splice_workspace_manifest(
             cargo_lockfile,
         ])
 
+    if skip_cargo_lockfile_overwrite:
+        arguments.extend([
+            "--skip-cargo-lockfile-overwrite",
+            "true",
+        ])
```

Comment:
Testing this fails with `error: unexpected argument 'true' found` and my guess would be here?